src/collections/components/CollectionProducts/keepProductOrder.ts

```diff
@@ -0,0 +1,23 @@
+import { CollectionProductsQuery } from "@dashboard/graphql";
+
+export type CollectionProducts = NonNullable<
+  NonNullable<CollectionProductsQuery["collection"]>["products"]
+>;
+
+/*
+  The API needs to take items in the same order as they came, and the reversed one when we move items down (negative position).
+  This function is designed to keep that order.
```

Comment:
"and the reversed one when we move items down"

I don't quite get this part - could you maybe add an example do this comment? 😅